basic_commands/seeking.md

```diff
@@ -37,3 +37,67 @@ If you want to inspect the result of a math expression, you can evaluate it usin
 
 
 In the visual mode you can press `u` (undo) or `U` (redo) inside the seek history to return back to previous or forward to the next location.
+
+## Open file
+
+As test file lets use some simple hello_world.c compiled in Linux ELF format.
+After we compiled it lets open it with radare2
+
+    r2 ./hello_world
+
+Now we have command prompt
+
+    [0x00400410]>
+
+Now we are ready to go deeper. 
+
+## Seeking at any position
+
+Seek to address 0x0
+
+    [0x00400410]> s 0x0
```

Comment:
All params could be in any base? Could be it described in some previous chapter as basics of using radare2?